src/components/link/Link.stories.js

```diff
@@ -95,6 +95,9 @@ export const LinkWithIcon = {
     >
       ${args.unnamed}
       <kd-icon
+        role="img"
+        aria-label="Arrow right icon"
```

Comment:
Alternatively, we could just use `aria-hidden` instead, if it doesn't add any value to describe the icon.

Reply:
It is necessary to readout icon at voiceover.

Reply:
Yes I know, I'm suggesting we might want to ignore it instead of read it.